src/components/BlogForm/NewPost.tsx

```diff
@@ -1,51 +1,43 @@
 "use client"; // needed for useform
 import { Button, Stack, TextField, Typography } from "@mui/material";
-import { useForm, Controller } from "react-hook-form";
-import { postBlog } from "../../api/blog";
 import { useSession } from "next-auth/react";
-import { DataProps } from "./types";
-import { enqueueSnackbar, useSnackbar } from "notistack";
-import { useEffect } from "react";
+import { useSnackbar } from "notistack";
+import { Controller, useForm } from "react-hook-form";
+import { postBlog } from "../../api/blog";
```

Comment:
use alias imports, e.g. @/api/blog

helps to prevent ugly ../../../ paths which can get quite long and confusing with big nested files

Reply:
for same folder imports thats not a problem though, i.e. import Type from ./types is fine